Gemfile

```diff
@@ -8,6 +8,8 @@ gem "chef-config", path: "chef-config" if File.exist?(File.expand_path("../chef-
 # Ensure that we can always install rake, regardless of gem groups
 gem "rake"
 
+gem "ffi", github: "lamont-granquist/ffi", branch: "lcg/rb_gc_guard_ptr" if RUBY_PLATFORM.downcase =~ /aix/
```

Comment:
master of ffi is busted on windows right now (and it might require checking the sources out and building with the custom rake task for windows, not sure...)